### PR TITLE
Deprecate old Telegraf chart 

### PR DESCRIPTION
**Pull request by jackzampolin**
This PR completes the breaking up of the telegraf chart by deprecating the current one.

Related PRs:
- https://github.com/kubernetes/charts/pull/822
- https://github.com/kubernetes/charts/pull/742

### CODE REVIEW

File: stable/telegraf/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## ref: https://hub.docker.com/r/library/telegraf/tags/
 image:
   repo: "telegraf"
-  tag: "1.1.0-alpine"
+  tag: "1.2.0-alpine"
```
Comment (prydonius) [inline stable/telegraf/values.yaml]: I would not bump the version when deprecating

File: stable/telegraf/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: telegraf
-version: 0.1.1
+version: 0.2.1
```
Comment (prydonius) [inline stable/telegraf/Chart.yaml]: 0.1.2 is probably a better bump since there should be no functional change in a deprecation

**Review by prydonius — CHANGES_REQUESTED** [review]
thanks @jackzampolin

**Comment by jackzampolin** [comment]
@prydonius Fixed that!

**Comment by abraithwaite** [comment]
This was merged but of the other two PRs, one was closed as stale and one is still open. 

What's the current status?

**Pull request merged**